### DISCUSSION TITLE
Find a better CSS methodology for allowing component consumers to customize component's layouts

### DISCUSSION THREAD

**Discussion by marinaaisa** [Qiskit/qiskit.org#2391] [General]
Find a methodology that allow us to balance between scoped styles and being able to modify child elements from their parents in CSS.

This issue is created because we should be able to find a better solution for `.copy-container:not(:only-child)` (check the [context](https://github.com/qiskit-community/community.qiskit.org/pull/99/files#diff-48b78652ea3a1dcef370efb835a0baa3R57) and [problem](https://github.com/qiskit-community/community.qiskit.org/pull/99/files#r338023680)) and also avoid [this kind of approach that I suggested](https://github.com/qiskit-community/community.qiskit.org/pull/99#discussion_r338030743) since it's following an anti-pattern called coupling. Thanks for the feedback, @delapuente 

Don't hesitate to contact me if this issue is not properly explained.

**Comment by delapuente** [comment]
Let's try BEM without removing `scoped` when possible and being flexible with the inner hierarchies, trying not to over-componentize.
